internal/commands/release_notes_test.go

```diff
@@ -72,7 +72,7 @@ func TestReleaseNotes_Execute(t *testing.T) {
 			repoOwner:  "bunch",
 			repoName:   "banana",
 			readFile:   readFileFunc,
-			fetchNotesData: func(c context.Context, repo *git.Repository, ghc *github.Client, tro, trn, kfp, ir, fr string, iq notes.IssuesQuery) (notes.Data, error) {
+			fetchNotesData: func(c context.Context, repo *git.Repository, ghc *github.Client, tro, trn, kfp, ir, fr string, iq notes.IssuesQuery, tf notes.TrainstatFetcher) (notes.Data, error) {
```

Comment:
I might suggest renaming these arguments to be longer, such as using trainstatClient instead of tf, like the above change

Reply:
You might be able to get away with "_" instead of "tf" or "trainstatClient". I don't see any usages of that parameter in this helper function.

Reply:
Thank you both, I changed the name to "_" since as Chris pointed out that value is not being used in the function.